front50-core/src/main/groovy/com/netflix/spinnaker/front50/config/StorageServiceConfigurationProperties.java

```diff
@@ -0,0 +1,118 @@
+package com.netflix.spinnaker.front50.config;
+
+import java.util.concurrent.TimeUnit;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+
+@ConfigurationProperties("storage-service")
+public class StorageServiceConfigurationProperties {
+
+  private PerObjectType application = new PerObjectType(20, TimeUnit.MINUTES.toMillis(1));
+  private PerObjectType applicationPermission = new PerObjectType(20, TimeUnit.MINUTES.toMillis(1));
+  private PerObjectType serviceAccount = new PerObjectType(20, TimeUnit.MINUTES.toMillis(1));
+  private PerObjectType project = new PerObjectType(20, TimeUnit.MINUTES.toMillis(1));
+  private PerObjectType notification = new PerObjectType(20, TimeUnit.MINUTES.toMillis(1));
+  private PerObjectType pipelineStrategy = new PerObjectType(20, TimeUnit.MINUTES.toMillis(1));
+  private PerObjectType pipeline = new PerObjectType(20, TimeUnit.MINUTES.toMillis(1));
+  private PerObjectType pipelineTemplate = new PerObjectType(20, TimeUnit.MINUTES.toMillis(1));
+  private PerObjectType snapshot = new PerObjectType(2, TimeUnit.MINUTES.toMillis(1));
+  private PerObjectType deliveryConfig = new PerObjectType(20, TimeUnit.MINUTES.toMillis(1));
+  private PerObjectType pluginInfo = new PerObjectType(2, TimeUnit.MINUTES.toMillis(1));
+  private PerObjectType entityTags = new PerObjectType(2, TimeUnit.MINUTES.toMillis(5), false);
+
+  public PerObjectType getApplication() {
```

Comment:
do all these things need setters as well for config props binding or do the current values get bound values mapped onto them? (I honestly don't know mostly just asking for a friend, who is me)

Reply:
Good question, I'm not sure, so I added setters for them.